crates/sui-types/src/deny_list_v2.rs

```diff
@@ -72,10 +72,15 @@ impl MoveTypeTagTrait for AddressKey {
 }
 
 /// Rust representation of the Move type 0x2::deny_list::GlobalPauseKey.
+/// There is no u8 in the Move definition, however empty structs in Move
+/// are represented as a single byte 0 in the serialized data.
 #[derive(Debug, Serialize, Deserialize, Clone)]
-struct GlobalPauseKey;
+struct GlobalPauseKey(u8);
```

Comment:
same BCS encoding, but this one better captures what is going on here
```suggestion
struct GlobalPauseKey(bool);
```